lib/trino-parquet/src/main/java/io/trino/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -213,6 +213,9 @@ public static Domain getDomain(
         }
     }
 
+    /**
+     * {@code minimums} and {@code maximums} are equal-length arrays, with each pair of elements being the lower and upper bound of a range, respectively.
```

Comment:
nit: First sentence of javadoc should define what the documented thing is/does.
Make it into an ordinary comment. You can place it before the `List<Object> minimums,` line.

Reply:
I adjusted the wording slighlty

Reply:
Thanks, current wording works for a javadoc perfectly.